PROTO_tests/tests/index_test.py

```diff
@@ -50,6 +50,27 @@ def test_multiindex_creation(self, size):
         with pytest.raises(ValueError):
             idx = ak.MultiIndex([ak.arange(size), ak.arange(size - 1)])
 
+    def test_memory_usage(self):
+        from arkouda.dtypes import BigInt
+        from arkouda.index import Index, MultiIndex
+
+        idx = Index(ak.cast(ak.array([1, 2, 3]), dt="bigint"))
+        assert idx.memory_usage() == 3 * BigInt.itemsize
+
+        n = 2000
```

Comment:
Everything looks good, I would just recommend changing this to parameterizing `n` using the `@pytest.mark.parametrize("size", pytest.prob_size)` that other PROTO tests use

Reply:
I made the changes.  Let me know what you think.